subsys/bluetooth/mesh/Kconfig

```diff
@@ -620,6 +641,21 @@ config BT_MESH_RELAY_RETRANSMIT_INTERVAL
 	  messages, in milliseconds. Can be changed through runtime
 	  configuration.
 
+config BT_MESH_RELAY_BUF_COUNT
```

Comment:
PR says add support for an extended advertiser. May be we should tune the NCS_SAMPLES_DEFAULT if BT_MESH is used to configure number of sets.

Reply:
Never mind.